src/Switch-Thorlabs_StepperMotorMinimal/main.py

```
@@ -0,0 +1,197 @@
+        """Initialize the device. This function is called only once at the start of the measurement."""
+        try:
+            status = self.rack.IsSettingsInitialized()
+            #print(f"Rack settings initialized: {status}")
+        except Exception as e:
+            print("Could not check if rack is initialized:", e)
+
+        if not self.stepper.IsSettingsInitialized():
+            try:
+                self.stepper.WaitForSettingsInitialized(5000)
+                #print("Settings initialized.")
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)